packages/runtime/runtime-definitions/src/summary.ts

```diff
@@ -352,19 +380,9 @@ export interface ITelemetryContext {
 	 * @deprecated This interface should only be used for instrumenting, not for attempting to read already-set telemetry data.
 	 *
 	 * @param prefix - unique prefix for this data (ex: "fluid:map:")
-	 * @param property - property name of the telemetry data being tracked (ex: "DirectoryCount")
 	 * @returns undefined if item not found
 	 */
 	get(prefix: string, property: string): TelemetryBaseEventPropertyType;
```

Comment:
As discussed in the other thread with Alex, this should be removed here.  Anyone internal who needs it should cast to the class.

That said, we can also learn from cases like the one that is using this, to think of other functionality the general-purpose public API would need.  e.g. we could consider adding an `increment` function or something at some point.